src/main/java/TaskList.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private final String SEP = "____________________________________________________________";
+    private final List<Task> TASKS;
+
+    public TaskList() {
+        this.TASKS = new ArrayList<>();
+    }
+
+    public void addTask(Task task) {
+        TASKS.add(task);
+    }
+
+    public void printTasks() {
+        System.out.println("Todo List:");
+        int counter = 1;
+        for (Task t: TASKS) {
+            System.out.print(counter + ". ");
+            t.printTask();
+            counter += 1;
+            System.out.println();
+        }
+        System.out.println(SEP);
+    }
+
+    public void markTask(int num)  {
```

Comment:
Similarly, the markTask and unmarkTask can perhaps be refactored into 1 whole method